validator_client/src/http_api/graffiti.rs

```diff
@@ -0,0 +1,81 @@
+use crate::validator_store::ValidatorStore;
+use bls::PublicKey;
+use slot_clock::SlotClock;
+use std::sync::Arc;
+use types::{graffiti::GraffitiString, EthSpec, Graffiti};
+
+pub async fn get_graffiti<T: 'static + SlotClock + Clone, E: EthSpec>(
+    validator_pubkey: PublicKey,
+    validator_store: Arc<ValidatorStore<T, E>>,
+) -> Result<Graffiti, warp::Rejection> {
+    let initialized_validators_rw_lock = validator_store.initialized_validators();
+    let initialized_validators = initialized_validators_rw_lock.read();
+    match initialized_validators.validator(&validator_pubkey.compress()) {
+        None => Err(warp_utils::reject::custom_not_found(
+            "The key was not found on the server".to_string(),
+        )),
+        Some(_) => {
+            let Some(graffiti) = initialized_validators.graffiti(&validator_pubkey.into()) else {
+                return Ok(Graffiti::default());
```

Comment:
i was under the assumption that `Graffiti::default` was the process-wide default. However it seems thats probably not the case

Reply:
The process wide default is available to the `http_api` via the `graffiti_flag_filter here:

https://github.com/sigp/lighthouse/blob/2e71290f13d45fe3365924050d47451128b029df/validator_client/src/http_api/mod.rs#L222